src/consensus/params.cpp

```diff
@@ -24,7 +24,7 @@ namespace Consensus {
             // But, (blossomActivationHeight - consensusParams.SubsidySlowStartShift()) / consensusParams.nPreBlossomSubsidyHalvingInterval
             // would need to be treated as a rational number in order for this to work.
             // Define scaledHalvings := halvings * consensusParams.nPostBlossomSubsidyHalvingInterval;
-            int64_t scaledHalvings = ((blossomActivationHeight - SubsidySlowStartShift()) * Consensus::BLOSSOM_POW_TARGET_SPACING_RATIO)
+            int64_t scaledHalvings = ((blossomActivationHeight - SubsidySlowStartShift()) * Consensus::BLOSSOM_POW_EXPECTED_SECS_PER_BLOCK_RATIO)
```

Comment:
Ditto, BlossomPoWTargetSpacingRatio is from the spec (section 7.6.3).

Reply:
Ack

Reply:
I've reverted those changes, per your request.

Reply:
I think this PR is ready to merge!